dataplane/burst.h

```diff
@@ -0,0 +1,125 @@
+#pragma once
+
+#include <rte_ethdev.h>
+#include <rte_mbuf.h>
+
+#include "common.h"
+#include "common/type.h"
+
+namespace dpdk
+{
+
+class BurstView
+{
+	rte_mbuf** first_;
+	uint16_t len_ = 0;
+
+public:
+	BurstView(rte_mbuf** first, uint16_t length) :
+	        first_{first}, len_{length}
+	{
+	}
+	bool empty() { return len_ == 0; }
```

Comment:
Here we should put `[[nodiscard]]` btw like the STL does.
(they have a valid reasoning, as new users tend to think that it's a verb, and that it will remove all the elements of a container).

Reply:
Good insight, was wondering why they marked it and not `size()`